docs/overview/excel.md

```diff
@@ -8,9 +8,9 @@ ms.localizationpriority: high
 
 # Office Scripts in Excel
 
-Office Scripts in Excel let you automate your day-to-day tasks. Inside Excel on the web, you can record your actions with the Action Recorder. This creates a TypeScript language script that can be run again any time. You can also create and edit scripts with the Code Editor. Your scripts can then be shared across your organization so your coworkers can also automate their workflows.
+Office Scripts in Excel let you automate your day-to-day tasks. You can create and edit scripts with the Code Editor. Run a series of Excel steps with a single button press. Then, share that script with coworkers so everyone can improve their workflow. Inside Excel on the web, you have an additional tool: the Action Recorder. This turns your manual steps into Office Scripts without having to look at any code.
```

Comment:
Actually, we shouldn't use either: [press guidelines](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29040), [click guidelines](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33540)

Per guidelines, we should use "select" or perhaps "selection" in this context: [Describing interactions with UI](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=26472)

I suggested removing "press" because it's not inclusive to screenreader users, but your comment and the guidelines note the other exclusion issues with my replacement word "click".

Reply:
Thanks Alison. Honestly, the sentence reads fine with no word there ("with a single button"), so I think I'll go that direction.

Reply:
Updated the term throughout the content set.